src/evaluators/interpolation/PHAL_DOFCellToSide.hpp

```diff
@@ -40,9 +40,12 @@ class DOFCellToSideBase : public PHX::EvaluatorWithBaseImpl<Traits>,
 private:
 
   std::string                     sideSetName;
-  std::vector<std::vector<int> >  sideNodes;
   std::vector<int>                dims;
 
+  Kokkos::DynRankView<int, PHX::Device> sideNodes;
```

Comment:
I think this one can safely be a `Kokkos::View`. If I remember correctly, DynRankView is required for AD for some reason but this is just used for indexing so it should be safe to change.